crates/hir_def/src/docs.rs

```diff
@@ -93,7 +94,7 @@ fn merge_doc_comments_and_attrs(
 ) -> Option<String> {
     match (doc_comment_text, doc_attr_text) {
         (Some(mut comment_text), Some(attr_text)) => {
-            comment_text.push_str("\n\n");
+            comment_text.push_str("\n");
```

Comment:
Nit: `.push('\n')`, perhaps with `.reserve`.

Reply:
Right didn't realize I could drop the str part there. The whole module will ideally disappear with #6750 anyways though 👍